packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -72,46 +67,44 @@ export const MarketTypeDetails = (isEU?: boolean): TMarketTypeDetails => ({
     all: {
         description:
             'Trade swap-free CFDs on MT5 with forex, stocks, stock indices, commodities, cryptocurrencies, ETFs and synthetic indices.',
-        icon: <SwapFreeMT5Icon />,
-        iconWithWidth: (width: number) => <SwapFreeMT5Icon width={width} />,
+        icon: <PlatformIcon icon='SwapFree' />,
+        iconWithWidth: (width: number) => <PlatformIcon icon='SwapFree' width={width} />,
         title: 'Swap-Free',
     },
     financial: {
         description: isEU
             ? 'This MFSA-regulated account offers CFDs on derived and financial instruments.'
             : 'This account offers CFDs on financial instruments.',
-        icon: isEU ? <FinancialEUMT5Icon /> : <FinancialMT5Icon />,
-        iconWithWidth: isEU
-            ? (width: number) => <FinancialEUMT5Icon width={width} />
-            : (width: number) => <FinancialMT5Icon width={width} />,
+        icon: <PlatformIcon icon={isEU ? 'CFDs' : 'Financial'} />,
+        iconWithWidth: (width: number) => <PlatformIcon icon={isEU ? 'CFDs' : 'Financial'} width={width} />,
         title: isEU ? 'CFDs' : 'Financial',
     },
     synthetic: {
         description: 'This account offers CFDs on derived instruments.',
-        icon: <DerivedMT5Icon />,
-        iconWithWidth: (width: number) => <DerivedMT5Icon width={width} />,
+        icon: <PlatformIcon icon='Derived' />,
+        iconWithWidth: (width: number) => <PlatformIcon icon='Derived' width={width} />,
         title: 'Derived',
     },
 });
 
 export const PlatformDetails = {
     ctrader: {
-        icon: <CTraderIcon />,
-        iconWithWidth: (width: number) => <CTraderIcon width={width} />,
+        icon: () => <PlatformIcon icon='CTrader' />,
```

Comment:
@thisyahlen-deriv `icon` originally was not a function, if you change it you have to change all places in the code when we use `icon` and change `.icon` to `.icon()`, but better just not to convert it to a function

Reply:
So far i dont see we are using it. We are only using iconWithWidth for PlatformDetails. 
I think its okay this way since we can rename it to be jsx element.

Reply:
The component isn't created until this function is called

Reply:
@thisyahlen-deriv in this case, if you are modifying icon to a function, in this case we don't need to use iconWithWidth, you can just delete it and add props for icon. Like this

icon: props => <PlatformIcon icon='CTrader' {...props} />

And pass there all props related to SVG component

Reply:
@thisyahlen-deriv @shontzu-deriv uses icon in her PR ([link](https://github.com/binary-com/deriv-app/pull/13010/files#:~:text=if%20(platform%20%3D%3D%3D%20PlatformDetails.mt5,typeof%20PlatformDetails%5D%3F.icon%3B))

Reply:
Alrighty then, ill just remove iconwithwidth and make the icon as the only one. I am just gonna have width for now to be strict, but if we need more we can add it in the future